toolkit/block-producer-metadata/pallet/src/mock.rs

```diff
@@ -57,6 +59,27 @@ impl frame_system::Config for Test {
 	type PostTransactions = ();
 }
 
+impl pallet_balances::Config for Test {
+	type MaxLocks = ConstU32<50>;
+	type MaxReserves = ();
+	type ReserveIdentifier = [u8; 8];
+	type Balance = Balance;
+	type RuntimeEvent = RuntimeEvent;
+	type DustRemoval = ();
+	type ExistentialDeposit = ConstU128<1>;
+	type AccountStore = System;
+	type WeightInfo = pallet_balances::weights::SubstrateWeight<Test>;
+	type FreezeIdentifier = ();
+	type MaxFreezes = ();
+	type RuntimeHoldReason = ();
+	type RuntimeFreezeReason = ();
```

Comment:
In latest polkadot-sdk we will have to set it to something meaningful, I know it is only a mock but probably we will have to adjust demo runtime as well, saying that in case you will hit some compilation errors (https://github.com/input-output-hk/partner-chains/commit/043205ccf369c962617c7669bdb27d5beb25c121)
```suggestion
	type RuntimeHoldReason = RuntimeHoldReason;
	type RuntimeFreezeReason = RuntimeFreezeReason;
```